src/index.ts

```diff
@@ -1,7 +1,10 @@
 import shopifyAuth from './auth';
+import Shopify from '@shopify/shopify-api';
 
 export default shopifyAuth;
 
 export * from './auth';
+export * from '@shopify/shopify-api';
```

Comment:
I'm not a fan of doing this, but it solves a specific problem: if the app itself is requiring `shopify-api`, it may end up using a different object of `Shopify.Context` then this lib, which means `koa-shopify-auth` wouldn't identify it as being initialized.

One option we have is to expose a method like `initializeShopify` which would look something like:

```ts
export function initializeShopify(context: Shopify.Context) {
  Shopify.Context.initialize(context);
}
```

which leverages the fact that a context object can be used to initialize itself.